packages/server/lib/html/non_proxied_error.html

```diff
@@ -14,7 +14,13 @@
         <div class="automation-message">
           <p>Whoops, we can't run your tests.</p>
           <div>
-            <p class="muted">This browser was not launched through Cypress. Tests cannot run.</p>
+            <p class="muted">Either the browser was launched by Cypress and your organization has a ProxyMode group policy set or you visited the Cypress HTTP proxy port outside of a Cypress browser.</p>
+          </div>
+          <div className='helper-line'>
+            <a className='helper-docs-link' href='https://on.cypress.io/error-messages#Whoops-we-can-t-run-your-tests' target='_blank' rel='noreferrer'>
+              <i className='fas fa-question-circle'></i>
+              {' Why am I seeing this message?'}
+            </a>
```

Comment:
@flotwig kind of a follow up to my last question, is there an elegant way to reduce duplication?

Reply:
I don't know if there is a good way to reduce the duplication for this, since in one case it's JSX and another it's HTML. I'm sure we could in some way, but I doubt it's worth the effort.